<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TxtToImg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grpbox_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>330</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Input file</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Output directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_chooseFile">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_chooseFolder">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_font">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
     <property name="icon">
      <iconset theme="font">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_font">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>201</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fontColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Font color</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_backgroundColor">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fontColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_backgroundColor">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_quality">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_quality">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>290</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>75</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbox_nl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Parse new lines</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_resy">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>330</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_resx">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>330</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_resolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Image size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpbox_preview">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>330</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Preview</string>
    </property>
    <widget class="QLabel" name="label_preview">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_psize">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>261</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="icon">
     <iconset theme="do"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>107</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset theme="undo">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionTestData"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset theme="folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="info">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTestData">
   <property name="text">
    <string>Test data</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="icon">
    <iconset theme="folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open folder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset theme="undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
